### PR TITLE
Fixed issue with dropdown background being height-limited

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Fixed issue with the dropdown background being height-limited

Before:
Screenshot 2021-06-18 at 3 26 30 PM

After: 
Screenshot 2021-06-18 at 3 25 50 PM

### CODE REVIEW

File: webapp/src/widgets/valueSelector.scss [hunk 1]
```diff
@@ -56,6 +56,12 @@
             }
         }
     }
+
+    div:nth-child(3) {
```
Comment (jespino) [inline webapp/src/widgets/valueSelector.scss:60]: This is flaky and is not how react-select expects you to change the styles, in this case the approach is to use the property `maxMenuHeight` (you can find it here: https://react-select.com/props#select-props), but for changing styles inside react select components normally you change the styles function generator that you can find in the `valueSelector.tsx` file.

You can see more information about modifying the styles in https://react-select.com/styles

**Review by jespino — CHANGES_REQUESTED** [review]
Not the right approach for react-select

File: webapp/src/widgets/valueSelector.scss (hunk 1 above)
Comment (harshilsharma63) [reply webapp/src/widgets/valueSelector.scss]: Done. I've fixed it in react-select's recommended way.

**Review by jespino — APPROVED** [review]
LGTM

**Pull request merged**